### ISSUE TITLE
Core breaking changes in v0.25.x versus v0.24.x (`import/no-unresolved`, `require-atomic-updates`, and others removed) 

### ISSUE THREAD

**Issue by niftylettuce** [xojs/xo#416]
There are at least two rules that have been completely removed in v0.25 versus v0.24, which really break the flow of linting my project with XO, as these are pretty huge changes to a linter config; I would have though this would be a major version bump - or at the least mentioned in the Releases tab.

* `import/no-unresolved`
* `require-atomic-updates`

Looks like it was dropped here (7 months ago) at https://github.com/xojs/xo/commit/f069a21310fb933acd885612fda092c7e2530cab.

How is it "buggy"?  I didn't gather much context, but I would love to learn more.

**Comment by sindresorhus** [comment]
> I would have though this would be a major version bump

Minor releases pre-1.0.0 are considered major.

> import/no-unresolved

`eslint-plugin-import` has been a huge pain for years. It's super buggy, consistently breaks stuff, and its maintenance has been spotty. I have been tempted to remove all the rules, but I limited myself to the worst ones.

> require-atomic-updates

Lots of reported issues and more I've discovered that I didn't even bother to report as the rule is unusable.

https://github.com/eslint/eslint/issues?utf8=%E2%9C%93&q=%22require-atomic-updates%22+
